src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -784,7 +787,9 @@ public int ProgramRunLoop()
 
 				Render();
 
-				CheckMessages();
+				++frameCount;
+				if (0 == (frameCount % MessageCheckFrame))
+					CheckMessages();

```

Comment:
I very much disagree with this change. This will cause the UI to become even slower to respond to user interaction than it already is, given it waits for a full emulator frame to complete (and now 5!).

Reply:
This would make https://github.com/TASEmulators/BizHawk/issues/2959 even worse than it is